web/src/message_list_view.ts

```diff
@@ -1396,6 +1396,13 @@ export class MessageListView {
             }
         }
 
+        const is_conversation_view = this.list.data.filter.is_conversation_view();
+        if (is_conversation_view) {
+            $(".focused-message-list").addClass("is-conversation-view");
+        } else {
+            $(".focused-message-list").removeClass("is-conversation-view");
+        }
```

Comment:
Does this logic belong in `update_current_message_list`? The way you have it, this code gets run every time we rerender, and that shouldn't be necessary.

Reply:
Moved the logic to `update_current_message_list` to avoid this.